### DISCUSSION TITLE
Generate a report possible?

### DISCUSSION THREAD

**Discussion by djpirra** [aquasecurity/tfsec#1966] [Help and suppports]
Hi everyone,
Is it possible to generate a report with all passed and failed points?

**Comment by fatihtokus** [comment]
Hi djpirra,

I think the following parameters do the trick (https://aquasecurity.github.io/tfsec/v1.28.1/guides/usage/) If you include ignored and passed checks as well then you will have a complete report in result output. 

![image](https://user-images.githubusercontent.com/19647150/212477504-daa7d508-920b-4504-b879-3172e85c89c4.png)
